docs/docs/adding-a-list-of-markdown-blog-posts.md

```diff
@@ -74,4 +74,23 @@ export const pageQuery = graphql`
 `;
 ```
 
-This should get you a page with your posts sorted by descending date. You can further customise the `frontmatter` and the page component to get desired effects!
+### Creating the `PostLink` component
+
+The only thing left to do is to add the `PostLink` component. Create a new file `post-link.js` in `src/components/` and add the following:
+
+```js
+import React from "react";
+import Link from "gatsby-link";
+
+const PostLink = ({ post }) => (
+  <div>
+    <Link to={post.frontmatter.path}>
+      {post.frontmatter.title} ({post.frontmatter.date})
+    </Link>
+  </div>
+);
+
+export default PostLink;
+```
+
+This should get you a page with your posts sorted by descending date. You can further customise the `frontmatter` and the page and `PostLink` components to get desired effects!
```

Comment:
Nitpick, since you asked for an English review: "get desired effects" should be "get your desired effects" or "get the desired effects".

Other than regionally variant spellings like "customise"/"customize" (where I don't know your style guide so can't comment), everything else looks good to me.

Reply:
👍 Thank you!